tools/buildsys/src/manifest.rs

```diff
@@ -33,6 +33,35 @@ url = "https://bar"
 sha512 = "123456"
 ```
 
+The `bundle-*` keys on `external-files` are a group of optional modifiers
+and are used to untar an upstream external file archive,
+vendor any dependent code, and bundle it back up to be consumed.
+All three options are required when bundling an archive's dependences.
+
+`bundle-modules` is a list of module "paradigms" the external-file should
+be vendored through. For example, if a project contains a go.mod and go.sum file,
+"go" may be entered to vendor the dependencies.
+Currently, only "go" is supported.
+
+`bundle-path` is the filepath _within_ the archive that contains the module.
+So, for example, given a go project that has the necessary go.mod and go.sum
+files within the archive located at the filepath a/b/c,
+this `bundle-path` value should be `a/b/c`.
```

Comment:
Can we default this to the first sub-directory in the archive, assuming the root level is otherwise empty?

Reply:
👍🏼 Now defaults to the first top level directory in the archive if `bundle-path` is not provided